MaxText/configs/v5e/inference/llama2_70b_v5e-16.yml

```diff
@@ -0,0 +1,36 @@
+base_config: "inference_jetstream.yml"
+
+# WIP
+# tensor = 8, autoregressive=2
+# per_device_batch_size=6
+# weight bf16, kv cache bf16
+# around 29ms per decode step
+
+model_name: "llama2-70b"
+sharding_strategy: "experimental"
+attention: 'dot_product' # Supported attention: autoselected, dot_product, flash, cudnn_flash_te
```

Comment:
NIT: cudnn_flash_te does not work for inference, and tpu flash is broken on 70b, let's just leave supported attention as dot_product until verified.

Reply:
Thanks for this! Forget to check and remove the comment copied from base yaml.

Reply:
done.